docs/_data/components/schemas/problem.yaml

```diff
@@ -0,0 +1,41 @@
+description: See [RFC9457](https://www.rfc-editor.org/rfc/rfc9457#appendix-A).
+type: object
+properties:
+  type:
+    allOf:
+      - $ref: ./uri-reference.yaml
+      - description: |
```

Comment:
Ah cool I didn't know you could do this.  I've been frustrated in a few spots when including refs in schema that I couldn't add a context-specific description.